material-overrides/assets/stylesheets/moonbeam.css

```diff
@@ -2014,3 +2015,148 @@ p.quick-ref-title {
   height: unset;
   padding: 1em 2em;
 }
+
+/* Cookie Consent Styling */
+.md-consent__overlay {
+  width: calc(97% - 1px);
+  z-index: 200;
+}
+
+.md-consent__inner {
+  z-index: 999;
+  background-color: transparent;
+}
```

Comment:
```suggestion
.md-consent__inner {
  z-index: 999;
  background-color: transparent;
  width: 97%;
}
```

The widget is closer to the right side of the screen than it is the left. It's because (by default) material sets the `.md-consent__inner` to `width: 100%;` - so we need to update it here to be 97%

<img width="1785" alt="Screenshot 2025-02-19 at 2 43 19 PM" src="https://github.com/user-attachments/assets/a0d75907-81c2-4c6e-9763-0a3cf9ca0f4b" />